terraform/README.md

```diff
@@ -0,0 +1,117 @@
+# **Terraform Folder README**
+
+---
+
+## **Overview**
+
+This folder contains the Terraform scripts that define the complete infrastructure for the Band Camp Tracker project. The infrastructure is designed to collect data from the BandCamp website and API, process it through an ETL pipeline, store it in a database, generate PDF reports, send alerts based on trending and popular music genres and provide a dashboard for users to interact with and visualize the data.
+The key components defined in the Terraform folder are AWS lambda functions for the  ETL pipeline, report generation which uses EventBridge and SES to send daily reports, the dashboard which is hosted on AWS Fargate and utilises a Load Balancer for scalability, alerts which utilise AWS lambda which triggers via EventBridge once per hour, Amazon RDS, Amazon S3, Amazon ECR and AWS security features.
```

Comment:
Sorry to be annoying but maybe we should split this section into bullet points so it's easier to understand and have 'Key Components' as a heading